src/utils/logger.ts

```diff
@@ -0,0 +1 @@
+export const log = (args: string) => console.debug(args);
```

Comment:
is this necessary?

Reply:
It would be nice to have a central logger in the app.
But this may not be the best solution.
Removed.